SECURITY.md

```diff
@@ -1,41 +0,0 @@
-<!-- BEGIN MICROSOFT SECURITY.MD V0.0.7 BLOCK -->
```

Comment:
why do we want to remove this?

Reply:
@koepalex, @jschweda, please add the file back as it is `main`. It is a corporate requirement to have this file.

Reply:
Added `SECURITY.md` from `main` branch.